assets/css/global.less

```diff
@@ -2065,9 +2065,9 @@ body .guides .guides-section-white {
 
 // Post Detail styles
 .post-content {
-  padding: 0;
   color: @guides-font;
   line-height: 1.38;
+  padding-bottom: 50px;
```

Comment:
Removing the `padding: 0` above will cause other changes on the page we might not want. Perhaps override solely the bottom padding?

Reply:
Okay